src/OrchardCore.Modules/OrchardCore.GitHub/Settings/GithubAuthenticationSettings.cs

```diff
@@ -1,12 +1,16 @@
 using Microsoft.AspNetCore.Http;
+using OrchardCore.Security;
 
 namespace OrchardCore.GitHub.Settings
 {
-    public class GitHubAuthenticationSettings
+    public class GitHubAuthenticationSettings : OAuthSettings
     {
         public string ClientID { get; set; }
+
```

Comment:
I don't think these new line breaks are needed. Same in `GoogleAuthenticationSettings`, `AzureADSettings`, `MicrosoftAccountSettings`, `MicrosoftAccountSettings`.

Reply:
It's good for clarity & readability

Reply:
There's a reason we don't do this in every single place though :).

Reply:
That's why I surrprised in many places in the code base ;)

Reply:
I still think these line breaks are not needed. It doesn't add any readability any more than adding line breaks between any two lines of code.

Reply:
Adding lines between properties and methods is a good practice I have seen. Even ASP.NET Core team uses it in the code base

Reply:
If the properties have anything else on them, like attributes, or documentation, or they are multi-line, then sure. But otherwise, why? Why don't we add line breaks between private fields of a class too?

Reply:
That's the code lines I know from years :)